### DISCUSSION TITLE
Limit for Site provisioning API

### DISCUSSION THREAD

**Discussion by nilooy** [plausible/analytics#3157] [Self-Hosted Support] [answered]
I have a self hosted instance and i am using api to create sites, in my app, user can have their own custom domain and each user will have a analytics dashboard. (Multi tenant)

<img width="1028" alt="Screenshot 2023-07-18 at 19 17 29" src="https://github.com/plausible/analytics/assets/32486682/dd493b35-381e-4873-b3ab-72af76f21c4d">

I love the project, it's amazing. i need help from the community for the following topic please.
i read in the docs that https://plausible.io/docs/sites-api that it has limits up to 50 sites, is it the case for self hosted one as well? if yes, how to remove the limit?

**Comment by HenkVanMaanen** [comment, marked as answer]
We have more than 1000 domains / websites on our self hosted instance, it's no problem. Just keep in mind you have to execute a query in the database to change the permission of the api key to allow using the site provisioning API. 

`UPDATE api_keys SET scopes = '{sites:provision:*,stats:read:*}';`

**Reply by nilooy** [reply]
ok perfect, thanks
